play-java-websocket-example/app/actors/UserParentActor.java

```diff
@@ -26,7 +26,7 @@ public Create(String id, ActorRef<Flow<JsonNode, JsonNode, NotUsed>> replyTo) {
 
     public static Behavior<Create> create(UserActor.Factory childFactory, Config config) {
         return Behaviors.setup(context -> {
-            Set<String>defaultStocks = new HashSet<>(config.getStringList("default.stocks"));
```

Comment:
TIL it's valid java to not add a blankspace between the type and the variable name. That is `Set<String>defaultStocks` was valid code and compiled successfully.